### ISSUE TITLE
Panic during shutdown: transaction verifier is always ready: Closed

### ISSUE THREAD

**Issue by rex4539** [ZcashFoundation/zebra#3053]
## Error
```
transaction verifier is always ready: Closed
```

## Metadata
|key|value|
|--|--|
|**version**|1.0.0-beta.0+30.g11b5a33|
|**Zcash network**|Mainnet|
|**state version**|10|
|**branch**|main|
|**git commit**|11b5a33|
|**commit timestamp**|2021-11-10T23:51:22+00:00|
|**target triple**|x86_64-apple-darwin|
|**build profile**|debug|
|**location**|/Users/rex/zebra/zebra-consensus/src/block.rs:184:22|

## SpanTrace

<details>

```
SpanTrace:
   0: zebrad::components::sync::downloads::download_and_verify
           with hash=00000000003c67ed4f4e6fb6dd80c258e03a38e5d9b0c8423b8a7019bf15fc16
             at zebrad/src/components/sync/downloads.rs:154
   1: zebrad::components::sync::extend_tips
             at zebrad/src/components/sync.rs:515
   2: zebrad::components::sync::sync
             at zebrad/src/components/sync.rs:285
   3: zebrad::application::
           with zebrad="11b5a33" net="Main"
             at zebrad/src/application.rs:363
```
</details>

## Backtrace

<details>

```
Backtrace:
   0: backtrace::backtrace::libunwind::trace
             at .cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.54/src/backtrace/libunwind.rs:90:5
      backtrace::backtrace::trace_unsynchronized
             at .cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.54/src/backtrace/mod.rs:66:5
   1: backtrace::backtrace::trace
             at .cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.54/src/backtrace/mod.rs:53:14
   2: backtrace::capture::Backtrace::create
             at .cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.54/src/capture.rs:176:9
   3: backtrace::capture::Backtrace::new
             at .cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.54/src/capture.rs:140:22
   4: color_eyre::config::PanicHook::panic_report
             at .cargo/registry/src/github.com-1ecc6299db9ec823/color-eyre-0.5.11/src/config.rs:982:18

```
</details>

Happened when I hit Ctrl+C to end this run -> https://www.dropbox.com/s/bhjx3v0xd6k36t0/debug.log.zip?dl=0

**Comment by teor2345** [comment]
This is a known issue, we're hoping to fix it in tickets #1677 or #1678.

**Comment by teor2345** [comment]
We haven't seen this bug for a long time, we think it's fixed.
